contracts/access/manager/AccessManager.sol

```diff
@@ -277,14 +277,14 @@ contract AccessManager is Context, Multicall, IAccessManager {
     }
 
     /**
-     * @dev Remove an account for a role, with immediate effect. If the sender is not in the role, this call has no
-     * effect.
+     * @dev Remove an account from a role, with immediate effect. If the account is not in the role, this call
+     * has no effect.
      *
      * Requirements:
      *
      * - the caller must be in the role's admins
      *
-     * Emits a {RoleRevoked} event
+     * Emits a {RoleRevoked} event if the account was in the role.
```

Comment:
`was in the role` or `had the role`.

To me an element is in a group, but a member has a role. Somehow a group is something you are in while a role is something attached to you. Does that make sens ?

Reply:
I see. That makes sense.

Reply:
I followed the wording that was in the rest of the docs but that makes sense.

Reply:
I fixed this, and changed a few other things I noticed missing in the docs.